### PR TITLE
Fix exos_config:save_when when a stopped process is present

### PR DESCRIPTION
**Pull request by uedvt359**
##### SUMMARY

NOTE: This is distinct from Bug #109, which I filed earler!

When a stopped process exists on the Switch, the prompt for the `save` command changes. This trips up `save_when`, which does not recognize the prompt and it runs into a timeout. 

commit message:

>When a process is created, and then terminated but not deleted, the
>save config prompt will change. This module expected the old prompt,
>which never arrived, so the save will run into a timeout.
>
>This is how the special prompt looks:
>> * Slot-1 HOSTNAMEHERE.1 # save
>> The configuration file primary.cfg already exists.
>> Some process has been terminated/stopped. If you save the configuration
>> then that process's configuration will be lost.
>> Do you want to continue? (y/N)

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`exos_config`

##### ADDITIONAL INFORMATION

```paste below
---
- hosts: all
  connection: network_cli
  user: uedvt359
  vars: 
    ansible_network_os: exos
    
  tasks:
  - name: create a process and make sure it was started at least once
    exos_config: 
      commands:
      - create process testProcess python-module __future__ start on-demand vr VR-Mgmt
      - start process testProcess
      save_when: changed

  - name: stop the saved process
    exos_config: 
      commands:
      - terminate process testProcess forceful

  - name: this will fail, since the prompt is unexpected
    exos_config:
      save_when: always

```

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
You need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

File: changelogs/fragments/110-fix-exos-save-config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - exos_config - Fix a hang due to an unexpected prompt during save_when. (https://github.com/ansible-collections/community.network/pull/110)
```
Comment (felixfontein) [inline changelogs/fragments/110-fix-exos-save-config.yaml:2]:
```suggestion
  - exos_config - fix a hang due to an unexpected prompt during save_when. (https://github.com/ansible-collections/community.network/pull/110).
```

**Comment by felixfontein** [comment]
CC @hlrichardson @rdvencioneck @LindsayHill @bigmstone @ujwalkomarla you are listed as authors/maintainers of this module. Please check this change.

File: changelogs/fragments/110-fix-exos-save-config.yaml (hunk 1 above)
Comment (uedvt359) [reply changelogs/fragments/110-fix-exos-save-config.yaml]: applied.

**Comment by dericcrago** [comment]
thank you for your contribution @uedvt359

**Pull request merged**